lisa/sut_orchestrator/ready.py

```diff
@@ -66,7 +83,10 @@ def _deploy_environment(self, environment: Environment, log: Logger) -> None:
 
     def _delete_environment(self, environment: Environment, log: Logger) -> None:
         # ready platform doesn't support delete environment
-        pass
+        if self._ready_runbook.ignore_dirty_env:
+            # if ignore_dirty_env is set, it will reset the environment to clean
+            # then test cases can be run on the same environment
+            environment.mark_clean()
```

Comment:
this happens before `environment.mark_dirty()`, will find a right place to do this.

Reply:
Try to overwrite the `deploy_environment` method like below in the ready platform.
```python
def deploy_environment(self, environment: Environment) -> None:
    #  check and overwrite dirty flag here
    super().deploy_environment(environment)
```

Reply:
this also happens before `environment.mark_dirty()`

![image](https://github.com/user-attachments/assets/58b15694-8d0e-418c-b394-67f9a46064f2)

Reply:
How about overwrite the delete_environment?

Reply:
current implement works